src/CMakeLists.txt

```diff
@@ -45,9 +45,27 @@ endif()
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
 
+target_include_directories(cucumber-cpp-nomain PUBLIC
+    $<BUILD_INTERFACE:${CUKE_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>
+    PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(cucumber-cpp PUBLIC
+    $<BUILD_INTERFACE:${CUKE_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>
+    PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR})
+
 if(MINGW)
     list(APPEND CUKE_DEP_LIBRARIES ws2_32)
 endif(MINGW)
 
 target_link_libraries(cucumber-cpp-nomain PRIVATE ${CUKE_DEP_LIBRARIES})
 target_link_libraries(cucumber-cpp PRIVATE ${CUKE_DEP_LIBRARIES})
+
+install(TARGETS cucumber-cpp-nomain
+    ARCHIVE  DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY  DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME  DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS cucumber-cpp
```

Comment:
Why the duplication? The `install()` command supports multiple targets after `TARGETS`.